docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -0,0 +1,118 @@
+# CREATE SKILL - NEW COMMAND
+
+`askx new` allows developers to create an `Alexa hosted skill`, and clone the skill project to the local machine, which replaces and optimizes `ask create-hosted-skill` command in CLI V1.
+
+Alexa hosted deployer builds Alexa skills quickly by provisioning required resources automatically. 
+Alexa hosted service will create a new AWS account with a deployment pipeline (CodeBuild), a skill Lambda function, a Git repository (CodeCommit) and an S3 bucket for developers.
+Thus, the developers can then develop on the editor inside the portal or checkout the code using Git and develop with their IDEs.
+
+**STRUCTURE OF INIT COMMAND:**
+
+`askx init [-p | --profile <profile>] [--debug] [-h | --help]`
+
+**OPTIONS DESCRIPTION:**
+
+**profile**: Optional. Specify a profile name to be used. Defaults to use `default` as the profile name, if this option or environmental variable `ASK_DEFAULT_PROFILE` is not set.
+
+**debug**: Optional. Appends a debug message to the standard error.
+
+
+## WORKFLOW:
```

Comment:
Do we need this section? Seems too much details.

Reply:
For hosted skill, it may be necessary because the flow and options are different from other deployers. For example, we have to let user know to select`@ask-cli/hosted-skill-deployer` to create a hosted skill.